vector/src/main/res/values/strings.xml

```diff
@@ -1400,4 +1402,10 @@ Why choose Riot.im?
     <string name="autodiscover_well_known_autofill_dialog_message">Riot detected a custom server configuration for your userId domain \"%s\":\n%s</string>
     <string name="autodiscover_well_known_autofill_confirm">Use Config</string>
 
+
+    <string name="title_activity_emoji_reaction_picker">Reactions</string>
```

Comment:
Please add new String in file string_riotX.xml

Reply:
Done